### PR TITLE
3 Rhyming Puzzles Added #14

### PR DESCRIPTION
**Pull request by wajeehsheikh**
Hi, 

As instructed, I added three new rhymes to the puzzles.js. I hope they are good enough to be merged. 
Thank you,
Wajeeh

### CODE REVIEW

File: src/js/puzzles.js [hunk 1]
```diff
@@ -314,10 +314,29 @@ var puzzlesArray = [
   word: 'cards',
   hint: 'unit of distance',
   answer: 'yards'
+},
+{
+  id: 46,
+  sentence: 'I left my boat at the rock',
+  word: 'rock',
+  hint: 'on water',
```
Comment (Arcaster42) [inline src/js/puzzles.js]: Can we get a slightly better hint? Perhaps something like "building on water" or "building for ships".

File: src/js/puzzles.js [hunk 2]
```diff
@@ -314,10 +314,29 @@ var puzzlesArray = [
...
+{
+  id: 47,
+  sentence: 'Everytime it gains, it pours',
+  word: 'gains',
+  hint: 'weather',
+  answer: 'rains'
+},
+{
+  id: 48,
+  sentence: 'Are you sure, this will be sun?',
```
Comment (Arcaster42) [inline src/js/puzzles.js]: No need for the comma here.

File: src/js/puzzles.js [hunk 3]
```diff
@@ -314,10 +314,29 @@ var puzzlesArray = [
...
+  word: 'sun',
+  hint: 'Having the best time of my life',
```
Comment (Arcaster42) [inline src/js/puzzles.js]: Superlative here seems a little strong. Maybe something like "having a good time".

File: src/js/puzzles.js (hunk 1 above)
Comment (wajeehsheikh) [reply src/js/puzzles.js]: About to send pr with changes

File: src/js/puzzles.js (hunk 2 above)
Comment (wajeehsheikh) [reply src/js/puzzles.js]: About to send pr with changes

File: src/js/puzzles.js (hunk 3 above)
Comment (wajeehsheikh) [reply src/js/puzzles.js]: About to send pr with changes

**Pull request merged**